example/lib/main.dart

```diff
@@ -91,7 +91,7 @@ class _MyAppState extends State<MyApp> {
   }
 
   void authenticate() async {
-    final url = 'http://localtest.me:43823/';
+    final url = 'http://127.0.0.1:43823/';
```

Comment:
Why was this changed? 🤔

Reply:
For me, `localtest.me` appearently doesn't work anymore (at least, it didn't work yesterday)

Reply:
Just checked again. `localtest.me` doesn't work in Android Emulators. Haven't tested on real devices yet.
However, since most testing is done on Android Emulators anyways, I would leave it as `localhost` now.
If you think, I should change it back, then I will do that :)
Otherwise, this PR is ready for merging